functions/exercise2/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+   padding: 20px;
+   background-color: #bdbdbd;
+}
+
+h1 {
+    text-align: center;
+    color: #330033;
+}
+
+input {
+    background-color: #ccccff;
+    text-align: center;
```

Comment:
Please do not use text-align center here. It does not allow to verify if you have spaces at the beginning.